inputs/Random.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace GameOfLife\inputs;
+
+use GameOfLife\Field;
+use Ulrichsg\Getopt;
+
+class Random extends BaseInput
+{
+    /**
+     * Add options
+     * Allows to add options to the variable $options in the gameoflife.php file.
+     * @param Getopt $_options
+     */
+    public function addOptions(Getopt &$_options)
+    {
+        $_options->addOptions
+        ([
+                ['f', "filling", Getopt::REQUIRED_ARGUMENT, "Allows to set the amount of living cells on the field."]
```

Comment:
If I read the description of the option I don't understand it. What is the amount? An absolute number like 10? A percentage?